batteries/dq_dv.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Jan  5 10:10:36 2023
+
+@author: mpopeil
+"""
+
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import sklearn as sk
+import sklearn.datasets
+from pandas import DataFrame
+from scipy.interpolate import Akima1DInterpolator
+import scipy
```

Comment:
It is better practice to import a package like this via an alias.  If this package has methods that share names with other methods, this will keep them unambiguous.  So now all scipy methods below can be used as `sp.[method_name]`

```suggestion
import scipy as sp
```